dotCMS/src/main/java/com/dotcms/rendering/velocity/services/ContentTypeLoader.java

```diff
@@ -0,0 +1,303 @@
+            Object contFieldValueObject = null;
+            if (UtilMethods.isSet(contField)) {
+                try {
+                    contFieldValueObject = field.name();
+                    contFieldValue = contFieldValueObject == null ? "" : contFieldValueObject.toString();
+                } catch (Exception e) {
+                    Logger.error(this.getClass(), "writeContentletToFile: " + e.getMessage());
+                }
+                if (!(field instanceof DateTimeField || field instanceof DateField || field instanceof TimeField)) {
+                    sb.append("#set( $")
+                        .append(field.variable())
+                        .append(" =\"")
+                        .append(UtilMethods.espaceForVelocity(contFieldValue)
+                            .trim())
+                        .append("\" )");
+                }
+            }
+            String fv = field.values() != null ? field.values() : "";
+            if (field instanceof TextField || field instanceof TextAreaField || field instanceof WysiwygField) {
+                sb.append("#set( $")
+                    .append(field.variable())
+                    .append(" =\"[ #fixBreaks($")
+                    .append(field.variable())
+                    .append(") ]\")");
+            } else if (field instanceof ImageField) {
+                // Identifier id = (Identifier) InodeFactory.getChildOfClassByRelationType(content,
+                // Identifier.class, field.getFieldRelationType());
+                String uri = "/html/images/shim.gif";
+                sb.append("#set( $")
+                    .append(field.variable())
+                    .append("ImageInode =\"")
+                    .append(Long.MAX_VALUE)
+                    .append("\" )");
+                sb.append("#set( $")
+                    .append(field.variable())
+                    .append("ImageWidth =\"")
+                    .append(150)
+                    .append("\" )"); // Original value was 165
+                sb.append("#set( $")
+                    .append(field.variable())
+                    .append("ImageHeight =\"")
+                    .append(150)
+                    .append("\" )"); // Originak value was 65
+                sb.append("#set( $")
+                    .append(field.variable())
+                    .append("ImageExtension =\"gif\" )");
+                sb.append("#set( $")
+                    .append(field.variable())
+                    .append("ImageURI =\"")
+                    .append(uri)
+                    .append("\" )");
+                sb.append("#set( $")
+                    .append(field.variable())
+                    .append("ImageTitle =\"[ Test Image Structure ]\" )");
+            } else if (field instanceof FileField) {
+                // Identifier id = (Identifier) InodeFactory.getChildOfClassByRelationType(content,
+                // Identifier.class, field.getFieldRelationType());
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)